ProcessSlave.php

```diff
@@ -384,18 +391,20 @@ protected function handleRequest(\React\Http\Request $request, HttpResponse $res
             );
             $this->shutdown();
         }
+        return $response;
     }
 
-    protected function prepareEnvironment(\React\Http\Request $request)
+    protected function prepareEnvironment(RequestInterface $request)
     {
         $_SERVER = $this->baseServer;
```

Comment:
> IMO this is bad when we enable concurrent requests

Mhhm. But even then we should add information to the PSR request, not to the SERVER vars?